tests/test_quantize.py

```diff
@@ -1 +1,25 @@
-# Last names: S
\ No newline at end of file
+# Last names: S
+import numpy as np
+from minecraft_gen import quantize
+
+def test_quantize():
+
+    # Simple single element test
+    assert quantize(np.array([0.0]), 1) == 0
+    assert quantize(np.array([10.0]), 3) == 2
+    assert quantize(np.array([-10.0]), 3) == 0
+
+    # Simple two element test
+    assert (quantize(np.array([-1,10.0]), 5) == np.array([0, 4])).all()
+    assert (quantize(np.array([-1,10.0]), 3) == np.array([0, 2])).all()
+
+    # Simple three element test
+    assert (quantize(np.array([-1,-1,10.0]), 2) == np.array([0, 0, 1])).all()
+    assert (quantize(np.array([-1,-1,10.0]), 3) == np.array([0, 0, 2])).all()
+    assert (quantize(np.array([-1,0,10.0]), 4) == np.array([0, 2, 3])).all()
+
+    # Test for output with known data 
+    gen = np.random.default_rng(42)
+    data = gen.random(10)*2-1
```

Comment:
```suggestion
    data = gen.uniform(0, 2, size=10)
```

Very minor preference, I think it reads a bit better. (If you do this, the numbers might need to be updated)